src/current/_includes/releases/v24.3/v24.3.0-alpha.1.md

```diff
@@ -0,0 +1,3 @@
+## v24.2.0-alpha.1
```

Comment:
I think this also will cause Kathryn some manual work generating the real alpha.1 release notes. Is it needed if the YAML entry is also omitted?

Reply:
Same as other comment.